<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="RAJU" failures="1" tests="2" name="TestRunner-FireFox" time="48.209" errors="0" timestamp="2024-09-24T18:04:00 IST">
  <testcase classname="com.practice1.CreateOrganizationTest" name="createOrganization" time="0.016">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.practice1.CreateOrganizationTest.createOrganization(CreateOrganizationTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- createOrganization -->
  <testcase classname="com.practice1.CreateOrganizationTest" name="createOrganizationVerifyPhoneNo" time="15.86"/>
</testsuite> <!-- TestRunner-FireFox -->
